pages/hold/request/[id].tsx

```diff
@@ -0,0 +1,240 @@
+import Head from "next/head"
+import {
+  Heading,
+  List,
+  Form,
+  FormField,
+  RadioGroup,
+  Radio,
+  ButtonGroup,
+  Button,
+} from "@nypl/design-system-react-components"
+
+import Layout from "../../../src/components/Layout/Layout"
+import {
+  PlainTextElement,
+  LinkedDetailElement,
+} from "../../../src/components/BibPage/BibDetail"
+import { SITE_NAME, BASE_URL, PATHS } from "../../../src/config/constants"
+
+import { findItemInBibResult } from "../../../src/utils/bibUtils"
+import { fetchBib } from "../../../src/server/api/bib"
+import initializePatronTokenAuth, {
+  doRedirectBasedOnNyplAccountRedirects,
+  getLoginRedirect,
+} from "../../../src/server/auth"
+import { getPatronData } from "../../api/account/[id]"
+
+import Bib from "../../../src/models/Bib"
+import Item from "../../../src/models/Item"
+
+import bibDetailStyles from "../../../styles/components/BibDetails.module.scss"
+
+import type { DiscoveryBibResult } from "../../../src/types/bibTypes"
+import type { DiscoveryItemResult } from "../../../src/types/itemTypes"
+
+interface HoldRequestPropsType {
+  discoveryBibResult: DiscoveryBibResult
+  discoveryItemResult: DiscoveryItemResult
+  isAuthenticated?: boolean
+}
+
+/**
+ * The Hold Request page renders a form to place a hold on an item.
+ * It renders some basic info about the bib and item based on the bib and item ids in the url,
+ * including pickup location options rendered as radio buttons in the form.
+ */
+export default function HoldRequestPage({
+  discoveryBibResult,
+  discoveryItemResult,
+  isAuthenticated,
+}: HoldRequestPropsType) {
+  const metadataTitle = `Item Request | ${SITE_NAME}`
+
+  const bib = new Bib(discoveryBibResult)
+  const item = new Item(discoveryItemResult, bib)
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    console.log("submitting form")
+  }
+
+  return (
+    <>
+      <Head>
+        <meta property="og:title" content={metadataTitle} key="og-title" />
+        <meta
+          property="og:site_name"
+          content={metadataTitle}
+          key="og-site-name"
+        />
+        <meta name="twitter:title" content={metadataTitle} key="tw-title" />
+        <title key="main-title">{metadataTitle}</title>
+      </Head>
+      <Layout isAuthenticated={isAuthenticated} activePage="search">
```

Comment:
There should be a new `activePage` value for the hold pages. Based on the designs, the hold pages don't have the search bar.

Reply:
Good call, i added this because I saw "Search" was active in the subnav, but forgot that this adds the search bar implicitly. Added a new hold type.